ttnn/tutorials/006.ipynb

```diff
@@ -794,12 +1034,14 @@
                     "name": "stdout",
                     "output_type": "stream",
                     "text": [
-                        "\u001b[38;2;000;128;000m                  Metal\u001b[0m | \u001b[1m\u001b[38;2;100;149;237mINFO    \u001b[0m | Closing device 0\n"
+                        "\u001b[38;2;000;128;000m                  Metal\u001b[0m | \u001b[1m\u001b[38;2;100;149;237mINFO    \u001b[0m | Closing device 0\n",
+                        "\u001b[38;2;000;128;000m                  Metal\u001b[0m | \u001b[1m\u001b[38;2;100;149;237mINFO    \u001b[0m | Disabling and clearing program cache on device 0\n"
                     ]
                 }
             ],
             "source": [
-                "ttnn.close_device(device)"
+                "   ttnn.synchronize_device(device)\n",
+                "   ttnn.close_device(device)"
```

Comment:
Why do we synchronize the device before closing it here?

Reply:
I have removed it.